torch/jit/__init__.py

```diff
@@ -996,10 +1041,17 @@ def script(obj, optimize=None, _frames_up=0, _rcb=None):
     r"""
     Scripting a function or ``nn.Module`` will inspect the source code, compile
     it as TorchScript code using the TorchScript compiler, and return a ``ScriptModule`` or
-    ``torch._C.Function``.
+    ``torch._C.Function``. TorchScript itself is a subset of the Python language, so not all
+    features in Python work, but we provide enough functionality to compute on
+    tensors and do control-dependent operations. For a complete guide, see the
+    `TorchScript Language Reference`_.
```

Comment:
Redundant underscore in the end?

Reply:
The `View File` renders it as markdown, a trailing underscore in `rST` is a link (this one is automatically created from the header with this same name

Reply:
I saw `_` appended in other spots and I was always wondering what it was used for> 😄